### PR TITLE
Jg/hero list module

### PR DESCRIPTION
**Pull request by jaimeglez**

### CODE REVIEW

File: src/components/Media.module.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import '../style/lib';
```
Comment (jaimeglez) [inline src/components/Media.module.scss:1]: I copied Media files from Marks' PR,

**Comment by luishdz1010** [comment]
@jaimeglez is this ready for review?

**Comment by jaimeglez** [comment]
> @jaimeglez is this ready for review?

Nope, I'm going to clean and add a spin off of this module as well

**Comment by jaimeglez** [comment]
Now it's ready to review

File: src/modules/HeroList.module.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@import '../style/lib';
+@import '../style/modules';
+
+.root {
+  @include module();
+  @include module-spacing-1();
+
+  .no-gutters {
```
Comment (luishdz1010) [inline src/modules/HeroList.module.scss:8]: No need to nest

File: src/modules/HeroFooterNav.module.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  .no-gutters {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+
+    // stylelint-disable-next-line plugin/selector-tag-no-without-class
+    > div {
+      padding: 0;
+    }
```
Comment (luishdz1010) [inline src/modules/HeroFooterNav.module.scss:16]: You can also pass a class to the items themselves, if that's not too hard

File: src/components/hero-overflow-text.module.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  .content-container {
+    padding: 0;
+    display: flex;
+    height: 100%;
+    position: relative;
+    z-index: $zindex-2;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+  }
```
Comment (luishdz1010) [inline src/components/hero-overflow-text.module.scss:66]: Is this nesting necessary?

**Review by luishdz1010 — APPROVED** [review]
Looks good. We need to start using real fields ASAP though

**Pull request merged**